src/main/kotlin/no/nav/sifinnsynapi/drift/DriftController.kt

```diff
@@ -0,0 +1,85 @@
+package no.nav.sifinnsynapi.drift
+
+import no.nav.k9.felles.log.audit.Auditdata
+import no.nav.k9.felles.log.audit.AuditdataHeader
+import no.nav.k9.felles.log.audit.CefField
+import no.nav.k9.felles.log.audit.CefFieldName
+import no.nav.k9.felles.log.audit.EventClassId
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import no.nav.security.token.support.core.api.RequiredIssuers
+import no.nav.security.token.support.core.context.TokenValidationContextHolder
+import no.nav.sifinnsynapi.Routes.SØKNAD
+import no.nav.sifinnsynapi.audit.Auditlogger
+import no.nav.sifinnsynapi.config.Issuers
+import no.nav.sifinnsynapi.oppslag.HentIdenterForespørsel
+import no.nav.sifinnsynapi.oppslag.IdentGruppe
+import no.nav.sifinnsynapi.oppslag.OppslagsService
+import no.nav.sifinnsynapi.soknad.DebugDTO
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus.OK
+import org.springframework.http.MediaType
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import java.time.LocalDateTime
+import java.time.ZoneOffset
+
+@RestController
+@RequiredIssuers(
+    ProtectedWithClaims(issuer = Issuers.AZURE)
+)
+class DriftController(
+    private val driftService: DriftService,
+    private val auditlogger: Auditlogger,
+    private val tokenValidationContextHolder: TokenValidationContextHolder,
+    private val oppslagsService: OppslagsService
+) {
+    companion object {
+        val logger = LoggerFactory.getLogger(DriftController::class.java)
+    }
+
+    @GetMapping("/debug$SØKNAD", produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
PostMapping her?

Burde kanske implementere en test som sjekker så att man ikke får tilgang utan tokens, med feil token, med token som ikke er validert osv? Kanske virker som forventet idag men en dependency bump down the road kanske brekker autentisering utan att vi merker noe?

Reply:
Parametrarna er jo sensitiv data, burde ikke sendes i get requests?

Reply:
Jo, det er et godt poeng.
Bra catch!

Reply:
Lagt til tester.